app/controllers/pdf_controller.rb

```diff
@@ -1,16 +1,22 @@
+# frozen_string_literal: true
+
 class PdfController < ApplicationController
   def create
     browser = Ferrum::Browser.new(timeout: 300, browser_options: { 'no-sandbox': nil })
     filename = "temp-#{DateTime.now.strftime('%Q')}"
-    File.open("tmp/#{filename}.html", "w") { |file| file.write(params[:content]) }
+    File.write("tmp/#{filename}.html", params[:content])
 
-    browser.go_to("file:///#{Rails.root}/tmp/#{filename}.html")
+    browser.go_to("file:///#{Rails.root.join("tmp/#{filename}.html")}")
     pdf_options = { path: "tmp/#{filename}.pdf" }
 
-    pdf_options[:landscape] = ActiveRecord::Type::Boolean.new.deserialize(params[:landscape]) if params[:landscape].present?
+    if params[:landscape].present?
+      pdf_options[:landscape] =
+        ActiveRecord::Type::Boolean.new.deserialize(params[:landscape])
+    end
+
     pdf_options[:scale] = ActiveRecord::Type::Float.new.deserialize(params[:scale]) if params[:scale].present?
```

Comment:
Let's change it to uniformity:

```
 if ...
	...
end
```

Reply:
Okay, however I have to disable the Rubocop `Style/IfUnlessModifier` for this line. Please check the [docs](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/IfUnlessModifier) for this rule to decide whether or not to use this check for the project.